objectivec/GPBWellKnownTypes.h

```diff
@@ -96,4 +115,82 @@ NS_ASSUME_NONNULL_BEGIN
 
 @end
 
+#pragma mark - GPBAny
+
+/**
+ * Category for GPBAny to help work with the message within the object.
+ **/
+@interface GPBAny (GBPWellKnownTypes)
+
+/**
+ * Returns a new instance of GPBAny that includes the serialized message. Uses
+ * the type.googleapis.com/ type_url prefix. On failure returns nil and fills
+ * in the optional errorPtr with the details. The error can be a
+ * GPBWellKnownTypesErrorDomain error or a general message serialization error.
+ */
++ (nullable instancetype)anyWithMessage:(nonnull GPBMessage *)message
+                                  error:(NSError **)errorPtr;
+
+/**
+ * Returns a new instance of GPBAny that includes the serialized message. On
+ * failure returns nil and fills in the optional errorPtr with the details. The
+ * error can be a GPBWellKnownTypesErrorDomain error or a general message
+ * serialization error.
+ */
++ (nullable instancetype)anyWithMessage:(nonnull GPBMessage *)message
+                          typeURLPrefix:(nonnull NSString *)typeURLPrefix
+                                  error:(NSError **)errorPtr;
+
+/**
+ * Serializes message and initializes the instance to contain the info. Uses
+ * the type.googleapis.com/ type_url prefix.  On failure returns nil and fills
+ * in the optional errorPtr with the details. The error can be a
+ * GPBWellKnownTypesErrorDomain error or a general message serialization error.
+ */
+- (nullable instancetype)initWithMessage:(nonnull GPBMessage *)message
+                                   error:(NSError **)errorPtr;
+
+/**
+ * Serializes message and initializes the instance to contain the info. On
+ * failure returns nil and fills in the optional errorPtr with the details. The
+ * error can be a GPBWellKnownTypesErrorDomain error or a general message
+ * serialization error.
+ */
+- (nullable instancetype)initWithMessage:(nonnull GPBMessage *)message
+                           typeURLPrefix:(nonnull NSString *)typeURLPrefix
+                                   error:(NSError **)errorPtr;
+
+/**
+ * Serializes message and sets the current instance to contain the info. Uses
+ * the type.googleapis.com/ type_url prefix. Returns success/failure, and on
+ * failure fills in the optional errorPtr with the details. The error can be a
+ * GPBWellKnownTypesErrorDomain error or a general message serialization error.
+ */
+- (BOOL)packWithMessage:(nonnull GPBMessage *)message
+                  error:(NSError **)errorPtr;
+
+/**
+ * Serializes message and sets the current instance to contain the info.
+ * Returns success/failure, and on failure fills in the optional errorPtr the
+ * details. The error can be a  GPBWellKnownTypesErrorDomain error or a general
+ * message serialization error (see GPBMessage).
+ */
+- (BOOL)packWithMessage:(nonnull GPBMessage *)message
```

Comment:
/**
Packs the serialized message into this GPBAny.

@ param message The message to be packed into the GPBAny.
@ param typeURLPrefix The URL prefix to apply for type_url.
@ param errorPtr Pointer to an error that will be populated if something goes wrong.

@ return Whether the packing was successful or not.
*/